### PR TITLE
Layout revisions

### PR DESCRIPTION
**Pull request by alexandercyu**
Changes layout so it is better for mobile viewing.

- The Scientific Experts and Learn More sections are on top of each other instead of side by side

### CODE REVIEW

File: static/css/daccs-project-css.css [hunk 1]
```diff
@@ -10,4 +10,9 @@ body{
 
 .logo-list-item{
     width: 25px;
+}
+
+.marble-github-link{
```
Comment (mishaschwartz) [inline static/css/daccs-project-css.css]: let's use "daccs" instead of "marble" here

File: templates/site/index.html [hunk 2]
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
 </div>
 {% endblock %}
 
+
+
+{% set table_data = [
+                {"name": "Steve Easterbrook", "institution": "University of Toronto", "department": "Computer Science"},
+                {"name": "Samuel Foucher", "institution": "Centre de recherche informatique de Montréal (CRIM)", "department": "Vision and Imaging Team"},
+                {"name": "Nathan Gillett", "institution": "Climate Research Division", "department": "Environment and Climate Change Canada"},
```
Comment (mishaschwartz) [inline templates/site/index.html]: as discussed: swap institution and department

File: templates/site/index.html [hunk 3]
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
...
+                {"name": "David Huard", "institution": "Ouranos", "department": "Scénarios et services climatiques"},
```
Comment (mishaschwartz) [inline templates/site/index.html]: as discussed: change to english translation

File: templates/site/index.html [hunk 4]
```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
...
+    <div class="row mt-4">
+     <div class="col-11 d-block mx-auto border rounded cover-text-box">
+      <h2>Scientific Experts</h2>
+        <div class="table-responsive">
+          <table class="table table-striped">
+           <thead>
+            <tr>
+                <th>Name</th>
+                <th>Institution</th>
+                <th>Department</th>
```
Comment (mishaschwartz) [inline templates/site/index.html]: as discussed: change to "Expertise"

File: static/css/daccs-project-css.css (hunk 1 above)
Comment (alexandercyu) [reply static/css/daccs-project-css.css]: Changed class to use "daccs"

File: templates/site/index.html (hunk 2 above)
Comment (alexandercyu) [reply templates/site/index.html]: Swapped institution and department for Nathan Gillett.

File: templates/site/index.html (hunk 4 above)
Comment (alexandercyu) [reply templates/site/index.html]: Changed to "Expertise"

**Review by mishaschwartz — APPROVED** [review]
Looks good. Are you planning on doing the other content changes (to the main paragraph) as part of this PR or in a different one?

**Pull request merged**